SemanticResultFormats.utils.php

```diff
@@ -71,4 +71,19 @@ public static function htmlQueryResultLink( $link ) {
 		return $link->getText( SMW_OUTPUT_HTML, $linker );
 	}
 
-}
\ No newline at end of file
+	/**
+	 * @since 3.2.0
+	 *
+	 * @param array $data
+	 *
+	 * @param string|null|bool $nonce
+	 *
+	 * @return string|WrappedString HTML
+	 */
+	public static function makeVariablesScript( $data, $nonce = null ) {
+		$script = ResourceLoader::makeConfigSetScript( $data );
+
+		return ResourceLoader::makeInlineScript( $script, $nonce );
```

Comment:
Are these static functions both available in MW 1.31?

Reply:
They are, much longer before that. ResourceLoader::makeConfigSetScript() is available since MW 1.17.  The former was introduced later in 2015, (that's around MW 1.26). Also note, these static methods are what Skin::makeVariablesScript is using under the hood without any fallback since MW 1.26.